webapp/src/errorList.tsx

```diff
@@ -108,4 +113,14 @@ export class ErrorList extends React.Component<ErrorListProps, ErrorListState> {
             exception
         })
     }
+
+    private generateStackTraces(exception: pxsim.DebuggerBreakpointMessage) {
+        return <div>
+            {(exception.stackframes || []).map(sf =>
+                <div className="stackframe">
+                    {pxt.Util.rlf("at {0} ({1}:{2}:{3})", sf.funcInfo.functionName, sf.funcInfo.fileName, sf.funcInfo.line + 1, sf.funcInfo.column + 1)}
```

Comment:
Use lf

Reply:
funcinfo guaranteed?

Reply:
It looks like it's a required property of the StackFrameInfo interface. Also, since funcInfo is of any type is it safe that access the properties like I'm doing right now?